pkg/net/net.go

```diff
@@ -83,72 +84,59 @@ func (o *Overlay) loop(ch <-chan EvtState, state event.Emitter) {
 }
 
 func (o *Overlay) gossipLoop() {
-	b := newBackoff(time.Millisecond*500, time.Minute, 2)
-	ticker := jitterbug.New(time.Millisecond*500, b)
+	var (
+		proto  = protoutil.Join(o.proto, "gossip")
+		bo     = newBackoff(time.Millisecond*500, time.Minute)
+		ticker = jitterbug.New(time.Millisecond*500, bo)
+	)
+
 	defer ticker.Stop()
 
 	for {
-		b.SetSteadyState(o.n.MaxLen() <= o.n.Len())
+		bo.SetSteadyState(o.n.MaxLen() <= o.n.Len())
 		select {
 		case <-o.proc.Closing():
 			return
 		case <-ticker.C:
 			p, err := o.n.RandPeer()
```

Comment:
Let's add a comment to the end of this line that says `// err is non-nil iff neighborhood is empty`.  It was non-obvious to us the first time, so it's likely to confuse someone else someday.